### DISCUSSION TITLE
MiKTex 编译失败

### DISCUSSION THREAD

**Discussion by frankn1ng** [sjtug/SJTUThesis#723] [提问] [answered]
试了一下用 VS Code 无法编译，运行 `Compile.bat` 也有报错，`main.log` 如下：
[main.log](https://github.com/sjtug/SJTUThesis/files/7482099/main.log)

求助，非常感谢！

**Comment by skyzh** [comment, marked as answer]
可以尝试先更新一下 MiKTeX 的包

**Reply by frankn1ng** [reply]
谢谢，确实可以了
